src/locales/zh.json

```diff
@@ -866,6 +886,23 @@
     "min": "Min",
     "open": "Open",
     "closed": "Closed",
+    "addLiquidity": "Add Liquidity",
+    "noLiquidity": "No liquidity found.",
+    "from": "From",
+    "to": "To",
+    "remove": "Remove",
+    "connectWallet": "Connect Wallet",
+    "approving": "Approving",
+    "approve": "Approve",
+    "approved": "Approved",
+    "output": "Output",
+    "amount": "Amount",
+    "removedLiquidity": "Removed Liquidity",
+    "removingLiquidity": "Removing Liquidity",
+    "earned": "Earned",
+    "poolRewards": "Pool Rewards",
+    "yourStake": "Your Stake",
+    "totalStake": "Total Stake",
     "notFound": "未找到"
   }
 }
```

Comment:
still there are mistake in zh locale. this file last line is 908 and en file last line is 916
and other file except zh is ending at 915
so there are still some issues.